src/scripts.js

```diff
@@ -1,47 +1,48 @@
-import users from './data/users-data';
-import recipeData from  './data/recipe-data';
-import ingredientsData from './data/ingredient-data';
-
-//API DATA
-import fetchedData from './fetch.js';
-
+// import users from './data/users-data';
+// import recipeData from  './data/recipe-data';
+// import ingredientsData from './data/ingredient-data';
 
+//CSS FILES
 import './css/base.scss';
 import './css/styles.scss';
-
+//API DATA
+import fetchedData from './fetch.js';
+//DOM UPDATES
 import domUpdates from './domUpdates.js';
-
+//IMAGES
 import './images/apple-logo-outline.png';
 import './images/apple-logo.png';
 import './images/cookbook.png';
 import './images/seasoning.png';
 import './images/search.png';
-
-
+//CLASSES
 import User from './user';
 import Recipe from './recipe';
-
+//QUERY SELECTORS
 let allRecipesBtn = document.querySelector(".show-all-btn");
 let filterBtn = document.querySelector(".filter-btn");
+let savedRecipesBtn = document.querySelector(".saved-recipes-btn");
 let fullRecipeInfo = document.querySelector(".recipe-instructions");
 let main = document.querySelector("main");
 // let menuOpen = false;
 // let pantryBtn = document.querySelector(".my-pantry-btn");
-let pantryInfo = [];
-let recipes = [];
-let favoriteRecipes; 
-let savedRecipesBtn = document.querySelector(".saved-recipes-btn");
 let searchBtn = document.querySelector(".search-btn");
 let searchForm = document.querySelector("#search");
 let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
 // let tagList = document.querySelector(".tag-list");
-let user;
 
+//GLOBAL VARIABLES
+let user;
+let usersData;
+let recipeData;
+let ingredientsData;
+let pantryInfo = [];
+let recipes = [];
+let favoriteRecipes;
```

Comment:
Maybe we should double check to see if any of these global variables can be local anywhere or called in a class or something?